forte/dmrg/block2_dmrg_solver.cc

```diff
@@ -737,7 +737,22 @@ Block2DMRGSolver::compute_complementary_H2caa_overlap(const std::vector<size_t>&
     const std::vector<int> tshape{na1, na1, na1};
     const std::vector<size_t> tstride{nactv * nactv, 1, nactv};
 
+    // reset stack memory
+    auto stack_mem =
+        static_cast<size_t>(dmrg_options_->get_double("BLOCK2_STACK_MEM") * 1024 * 1024 * 1024);
+    impl_->stack_mem_ = stack_mem;
+    
+    // read bond dimention
+    int bond_dim = 500;
+    auto sweep_bond_dims = dmrg_options_->get_int_list("BLOCK2_SWEEP_BOND_DIMS");
+    if (sweep_bond_dims.size() != 0)
+        bond_dim = sweep_bond_dims.back();
+    std::vector<block2::ubond_t> ket0_bond_dims(1, bond_dim);
+    std::vector<block2::ubond_t> bra_bond_dims(1, 2 * bond_dim);
+    std::vector<double> noises{0.0};
+
     // system initialization
+    auto integral_cutoff = 1.0e-14;
```

Comment:
should this be hardcoded or passed in as option?

Reply:
An option is added.